FutureNNAimbot/Program.cs

```diff
@@ -175,19 +188,18 @@ static void Main(string[] args)
                 }
                 if (yoloWrapper.DetectionSystem.ToString() != "GPU")
                 {
-                    MessageBox.Show("No GPU card detected. Exit...");
+                    MessageBox.Show("No GPU card detected. Exiting...");
                     Process.GetCurrentProcess().Kill();
                 }
                 objects = File.ReadAllLines($"trainfiles/{game}.names");
             }
             else
             {
                 trainingMode = true;
-                MessageBox.Show($"Looks like you haven't configured configs for game {game}... Let's go to train Neural Network! :)\n Prepairing files for training....");
+                MessageBox.Show($"Looks like you haven't configured settings for the game {game}... Let's train the Neural Network! :)\n Prepairing files for training....");
 
-                Console.Write("How much objects we will be analyse and train for detect? Write objects name via separator ',' without space : ");
+                Console.Write("How many objects will we be analyzing and training on? Write each object's name via separator ',' without space (EX: 1,2): ");
```

Comment:
How many objects will we be analyzing and training? Write each object's name via the separator ',' without space (EX: 1,2): ");

Reply:
Keeping "training on" because without "on", it sounds like we are training the objects rather than the NN.